Modules/xxlimited_35.c

```diff
@@ -247,43 +245,24 @@ xx_modexec(PyObject *m)
     /* Add some symbolic constants to the module */
     if (ErrorObject == NULL) {
         ErrorObject = PyErr_NewException("xxlimited_35.error", NULL, NULL);
-        if (ErrorObject == NULL) {
-            return -1;
-        }
     }
-    Py_INCREF(ErrorObject);
-    if (PyModule_AddObject(m, "error", ErrorObject) < 0) {
-        Py_DECREF(ErrorObject);
+    if (PyModule_AddObjectRef(m, "error", ErrorObject) < 0) {
```

Comment:
It is interesting, that the compiler complains about `PyModule_Add`, but not about `PyModule_AddObjectRef`. There is a bug in `PyModule_AddObjectRef` declaration.